eth-connector/src/lib.rs

```diff
@@ -418,7 +468,11 @@ impl EngineStorageManagement for EthConnectorContract {
 
     #[payable]
     fn engine_storage_unregister(&mut self, sender_id: AccountId, force: Option<bool>) -> bool {
-        self.assert_access_right().sdk_unwrap();
+        require!(
```

Comment:
Can you please use one method instead of duplicating this check?

Reply:
fixed: https://github.com/aurora-is-near/aurora-eth-connector/pull/71/commits/534375a95003affa591564bb05f791846cc03267